### ISSUE TITLE
Info and Relationships (WCAG 1.3.1h)

### ISSUE THREAD

**Issue by VivienDZB** [slidewiki/slidewiki-platform#1224]
**Request:** The intent of this Success Criterion is to ensure that information and relationships that are implied by visual or auditory formatting are preserved when the presentation format changes. For example, the presentation format changes when the content is read by a screen reader or when a user style sheet is substituted for the style sheet provided by the author.

**Issues:**

1. https://slidewiki.org/search?keywords=accessibility&sort=score

- The form fields within the "Advanced options" are not linked to their labels above. The corresponding form elements should be labeled with a label-element and be linked to the label above via aria-labeledby.

2. https://slidewiki.org/addDeck

- Labeling of the form fields "Wählen Sie das Thema" and "Wählen Sie die Tags" is not correctly linked to the associated input fields. The label and form field can be linked using aria-labeledby.

**Comment by VivienDZB** [comment]
### Search - https://slidewiki.org/search?keywords=accessibility&sort=score
The "Search" input field, as well as some form elements within the "Advanced Options", do not have `label` elements or are not properly linked to their labels.
### Add deck - https://slidewiki.org/addDeck
The „Wählen Sie das Thema“ and "Choose Tags" form elements are not properly linked to their labels. 
### Markdown editor - https://slidewiki.org/deck/131649-1/_/slide/856768-2/856768-2:2/markdownEdit
The text input field on this page, which is used to create or edit a presentation, does not have a `label` element.
### Contact us - https://slidewiki.org/contactus
The linking of labels and form elements has been implemented well overall on this page. Only the label “Type of report” does not seem to be linked to the associated form.

### How to fix it:
In order to create a link between the label and the form, either the `for` attribute is used when using `label`, or the labeled form field is included in the `label` element. If labels are not marked with the `label` element, it should be ensured that the associated form element is labeled in another way (e.g. via the `aria-labelledby` attribute).

**Comment by aoelen** [comment]
@rezatavakoli do you want to have a look at this issue?

**Comment by aoelen** [comment]
Closed in #1284
